### ISSUE TITLE
[sprint] Contributors Reward - Stickers

### ISSUE THREAD

**Issue by darol100** [hechoendrupal/DrupalConsole#2077]
### Problem / Motivation
We are trying to reward people that contribute to the Drupal Console projects by providing stickers base on what they have contributed too. Similar to what Drupal Core are doing with their stickers for sprints. Here is the list of the stickers that we have been thinking to create:
- **Developer**
 - The user that send new features or enhancement to project.
- **Translator**
 - For those users, that help with translating the project.
- **Documenter**
 - For those users that help, with the Drupal Console documentation.
- **Tester**
 - For those users that help testing Pull Request.

We need stickers  similar to the current DrupalConsole but with different colors and with the information about the achievement of the user. Here is an example of the Drupal Core contrib stickers - 
https://www.drupal.org/files/issues/sprint_role_stickers_d7_1.pdf

**Note:** Main goal of this stickers is to provide a reward to contributors this is not to replace the current drupal console logo. This idea was talking from Drupal Core, and here is the logic behind the stickers.....

### Why ?
TL;DR - Reward users for their contribution.
> IT workers are notoriously non-responsive to material rewards as incentives - the idea of competing for a "prize" can actually be a turn-off. Keep the rewards as nominal as possible. We are not here to earn "things". The act of completing a task was its own reward. On the other hand, achievements that involve bettering yourself, or earning recognition and kudos within the Drupal Community are often rated highly (while at the same time many IT folk are modest or shy, especially as first-timers), so make it visible. If it's a sticker or a badge, actually stick it to their shirt, bag or lanyard directly, don't give them something to hide in their pocket!

I have open this issue just to discuss the mocks and any other extra information about the stickers.

**Comment by jmolivas** [comment]
:+1:  on this @darol100 I love to bring those reward-stickers to DrupalCon.

**Comment by Saphyel** [comment]
Maybe can it start at drupal dev days milan?

**Comment by danielrose28** [comment]
Stickers look great. Wish I was going to DrupalCon now. ;)  I love the concept. Being a modest introvert myself, I'd rather just have a sticker to put on my laptop than being presented an award in front of a bunch of people.

**Comment by kabarca** [comment]
Proposal for Drupal Console badges. The example include badges for "Developer" and "Language Contributor". The idea is to change the color based on the badge type so it can be identified without having to read.

![badges](https://cloud.githubusercontent.com/assets/889027/14433840/2d3ee7a2-ffcd-11e5-829c-9fdbb83cb86a.jpg)

**Comment by darol100** [comment]
@kabarca , Great work. I think we should use either 2 or 6.

**Comment by kabarca** [comment]
I personally prefer #6 as well.

**Comment by omero** [comment]
@kabarca awesome work!!!!, is hard to decide but I'm going with 6 :smiley:

**Comment by danielrose28** [comment]
These look great. I like 5 and 6. Nice work. :+1:. Now I need to get off my butt and start contributing more. ;) This is such an awesome project overall. You guys make using console and contributing to it so much fun. Thank you.

**Comment by igasi** [comment]
Nice work guys!!!. I like 6.

**Comment by jmolivas** [comment]
@kabarca what will be the size of the stickers?

**Comment by Saphyel** [comment]
2 or 6

**Comment by kabarca** [comment]
@jmolivas I would suggest 3'x3' based on the Die Cut Stickers from StickerMule. I will work on getting the rest of the titles done.

![badges-approved](https://cloud.githubusercontent.com/assets/889027/14446535/57a153da-0014-11e6-8131-11710f1045b3.png)

**Comment by jmolivas** [comment]
I vote for number 6 as well.

**Comment by enzolutions** [comment]
I have vote for 1 or 4, I totally dislike 3

**Comment by jmolivas** [comment]
@kabarca May we try to keep one word per type how about `TRANSLATOR` instead of `LANGUAGE CONTRIBUTOR`

**Comment by dmouse** [comment]
vote for 1 :P

**Comment by david-hernandez** [comment]
A couple comments adding my two cents. (because Jesus asked for my feedback but probably doesn't realize the trouble that brings. ;) ) First, the designs look awesome. I wish I could get designers to make cool designs for my stuff.

I like 1, 2, and 6 the best because they have the most color. It makes the stickers easier to recognize from a distance. I somewhat prefer 1, since it keeps the well known console logo intact.

I would not make the stickers 3x3. That is quite large. Keep them smaller, so people are more likely to use them. Especially Drupal people who are fond of putting lots of stickers on their laptops and try not to make them overlap too much. Because of that I'm becoming more and more fond of smaller stickers. The mentor stickers (the stickers linked in the issue are not really core, they were made by the mentor program leads) are not overly large.

Also, I agree with Jesus' comment about wording. Keep it simple. Translator is better than language contributor. And I wouldn't break up the language contributors into two groups. Just leave it as one. I otherwise like the categories; developer, translator, documenter, tester. They are short enough that you could make the font larger.

Lastly, more of a question, is the intention to control the distribution and only give them to people as an actual reward? If so, I wouldn't do that. Don't hand them out like swag, but make sure they are freely available at events. And I wouldn't add a hierarchy to them like the core mentoring stickers have. That is one thing I don't like about them. (There is suppose to be a progression from one sticker to the next.)

**Comment by darol100** [comment]
> doesn't realize the trouble that brings. ;)

LOL

> like the categories; developer, translator, documenter, tester

This actually makes sense, updating the issue. 

> Lastly, more of a question, is the intention to control the distribution and only give them to people as an actual reward? 

We have not discuss about this, but is something we could try it out.

> And I wouldn't add a hierarchy to them like the core mentoring stickers have. That is one thing I don't like about them. 

I thought about it, but does not make sense with our contribution workflow. 

Thank you @david-hernandez  for you great feedback.

**Comment by kabarca** [comment]
Hi guys,

I have made a few changes to the badges.

1. Included all the set of badges
2. Barely noticeable but added the glare to the glasses so it looks more like the logo. The glare has the color of the badge according to the categories.
3. As you can see, only 4 badges and wording has changed!

![final-badges](https://cloud.githubusercontent.com/assets/889027/14591541/b9fc8482-04d8-11e6-9f36-23af0505a0c0.jpg)

**Comment by danielrose28** [comment]
@kabarca those look great! 👍

**Comment by kabarca** [comment]
Hey everyone, we should start wrapping up the topic so the stickers can be ready for DrupalCon NA.

**Comment by darol100** [comment]
@kabarca , I think the decision have been made number 6 see to be the winner. Anything else we need to dicuss ?

**Comment by kabarca** [comment]
Thank you all. Stickers are being printed and we will see them in DrupalCon!
